packages/twenty-server/src/modules/match-participant/match-participant.service.ts

```diff
@@ -111,6 +111,7 @@ export class MatchParticipantService<
 
     this.eventEmitter.emit(`${objectMetadataName}.matched`, {
       workspaceId,
+      name: `${objectMetadataName}.matched`,
```

Comment:
Why do we need the name now?

Reply:
we don't need it for this one indeed, but we need it for created, updated and deleted events because of a new check that was introduced in `workflow-trigger.service.ts` 
```
private async upsertEventListenerAndPublishVersion(
    workspaceId: string,
    workflowId: string,
    workflowVersionId: string,
    trigger: WorkflowDatabaseEventTrigger,
  ) {
    const eventName = trigger?.settings?.eventName;

    if (!eventName) {
      throw new WorkflowTriggerException(
        'No event name provided in database event trigger',
        WorkflowTriggerExceptionCode.INVALID_WORKFLOW_TRIGGER,
      );
    }
...
```